extensions/gdx-setup/src/com/badlogic/gdx/setup/resources/android/build.gradle

```diff
@@ -97,7 +97,7 @@ eclipse {
     }
 
     classpath {
-        plusConfigurations += [ project.configurations.compile ]        
+        plusConfigurations += [ project.configurations.implementation ]        
```

Comment:
For IDE setup (here `eclipse` and `idea`) you'll want to use the `compileClasspath` configuration that is resolvable, it'll include dependencies from `implementation`. See https://docs.gradle.org/current/userguide/java_plugin.html#tab:configurations